src/util/mapbox.js

```diff
@@ -100,18 +101,49 @@ export const normalizeTileURL = function(tileURL: string, sourceURL?: ?string, t
     const suffix = browser.devicePixelRatio >= 2 || tileSize === 512 ? '@2x' : '';
     const extension = browser.supportsWebp ? '.webp' : '$1';
     urlObject.path = urlObject.path.replace(imageExtensionRe, `${suffix}${extension}`);
+    urlObject.path = `/v4${urlObject.path}`;
 
-    replaceTempAccessToken(urlObject.params);
-    return formatUrl(urlObject);
+    return makeAPIURL(urlObject);
 };
 
-function replaceTempAccessToken(params: Array<string>) {
-    for (let i = 0; i < params.length; i++) {
-        if (params[i].indexOf('access_token=tk.') === 0) {
-            params[i] = `access_token=${config.ACCESS_TOKEN || ''}`;
+export const canonicalizeTileURL = function(url: string) {
```

Comment:
This can be an arrow function I think. Same in line 135.

Reply:
following convention here.

Reply:
👍 I thought maybe that was the case. Good to know.

Reply:
Nit: I think the convention is `export function fn(...)`, not `export const fn = function(...)`. So this looks like a pre-existing omission.

Reply:
yeah I was following convention of the functions in this file – do you want me to change all of them ?